.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tests/integration/fixtures"]
```

Comment:
Hey, you look like a guy who knows submodules (I dont :laughing: ) - could you show me how to add GDrive scripts as a submodule? And how to use them after that?

Reply:
I found this to be a decent introduction https://www.atlassian.com/git/tutorials/git-submodule

Git submodules _sound_ awesome when you learn about them the first time, but be cautioned: they have a few rough edges and _most_ of the time there's a better choice. For example having the GDrive scripts directly in this repo doesn't hurt, I think. Or a completely different repo which is only linked in the `README` if you want to decouple them from this repo.

Reply:
Yeah, last time I tried, they very *very problematic* (because of course I didn't read any tutorials :100:)

Huh - could we actually remove a test repo from submodules too, and just tell people to `git clone` it? We could just add a tag there, so it would be nicely versioned when CI/tests change

Reply:
Yes, we could. But I don't think we'd gain anything. The test code by itself is very small and wont' impact the repo size noticeably. 

On the other hand having tests and implementation split over two repos and thus two PRs makes reviewing _a lot_ harder. Often PRs change both (implementation changes, tests added) and not being able to discuss both in the same place is going to be cumbersome and annoying.

Reply:
No no, test repo meaning repo with test *images*